packages/docutils/lib/cli/index.ts

```diff
@@ -70,12 +71,34 @@ export async function main(argv = hideBin(process.argv)) {
        * Custom failure handler so we can log nicely.
        */
       (msg: string | null, error) => {
+        /**
+         * yargs' default output if an "implication" fails (e.g., arg _A_ requires arg _B_) leaves much to be desired.
+         *
+         * @remarks Unfortunately, we do not have access to the parsed arguments object, since it may have failed parsing.
+         * @param msg Implication failure message
+         * @returns Whether the message was an implication failure
+         */
+        const handleImplicationFailure = (msg: string | null): boolean => {
+          let match: RegExpMatchArray | null | undefined;
+          if ((match = msg?.match(IMPLICATIONS_FAILED_REGEX))) {
+            const [, arg, missingArg] = match;
```

Comment:
I would rather flip this condition and make it to a guard

Reply:
I don't understand what you're suggesting

Reply:
like
```js
if (!match) {
  return false;
}

const [, arg, missingArg] = match
```

avoids some nesting.

Reply:
done